src/qa4sm_reader/plotter.py

```diff
@@ -140,7 +140,7 @@ def _get_parts_name(Var, type='boxplot_basic') -> list:
         parts: list of parts for title
         """
         parts = []
-        ref, mds, other = [meta for meta in Var.get_varmeta()]
+        ref, mds, other, _ = [meta for meta in Var.get_varmeta()]
```

Comment:
I know it is not originated in your code, but can't this simply be:

`ref, mds, other, _ = Var.get_varmeta()` ?

Reply:
yes its the same (even though [meta for meta in Var.get_varmeta()] is a list)